### PR TITLE
feature(ui-ux): assets sort row and modal 2.0

### PR DESCRIPTION
**Pull request by siradji**
#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx [hunk 1]
```diff
@@ -567,9 +590,9 @@ export function PortfolioScreen ({ navigation }: Props): JSX.Element {
   )
 }
 
-function AssetSortRow (props: { hideIcon: boolean, isSorted: boolean, assetSortType: PortfolioSortType, modifiedDenominationCurrency: string, onPress: () => void }): JSX.Element {
-  const highestCurrencyValue = translate('screens/PortfolioScreen', 'Highest {{modifiedDenominationCurrency}} value', { modifiedDenominationCurrency: props.modifiedDenominationCurrency })
-  const lowestCurrencyValue = translate('screens/PortfolioScreen', 'Lowest {{modifiedDenominationCurrency}} value', { modifiedDenominationCurrency: props.modifiedDenominationCurrency })
+function AssetSortRow (props: { isSorted: boolean, assetSortType: PortfolioSortType, modifiedDenominationCurrency: string, onPress: () => void }): JSX.Element {
+  const highestCurrencyValue = translate('screens/PortfolioScreen', 'Highest value ({{modifiedDenominationCurrency}})', { modifiedDenominationCurrency: props.modifiedDenominationCurrency })
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx:594]: update translations too

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/BottomSheetAssetSortList.tsx [hunk 2]
```diff
@@ -56,45 +55,30 @@ export const BottomSheetAssetSortList = ({
     index: number
   }): JSX.Element => {
     return (
-      <ThemedTouchableOpacity
-        style={tailwind('px-4 py-3 flex-row justify-between')}
+      <ThemedTouchableOpacityV2
+        dark={tailwind('border-mono-dark-v2-300')}
+        light={tailwind('border-mono-light-v2-300')}
+        style={tailwind('px-5 py-3 flex-row  items-center justify-between border-b-0.5 py-2.5', { 'border-t-0.5': index === 0 })}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/BottomSheetAssetSortList.tsx:61]: should move `px-5` to container, noticed the border bottom is not full width

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/BottomSheetAssetSortList.tsx [hunk 3]
```diff
@@ -56,45 +55,30 @@ export const BottomSheetAssetSortList = ({
...
+          style={tailwind('py-2 text-sm font-normal-v2')}
+        >
           {translate('screens/PortfolioScreen', getDisplayedSortText(item))}
-        </ThemedText>
+        </ThemedTextV2>
         {selectedAssetSortType === item && (
           <ThemedIcon
-            size={24}
-            name='check'
+            size={18}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/BottomSheetAssetSortList.tsx:75]: to compensate icon's "internal padding"
```suggestion
            size={20}
```

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx [hunk 4]
```diff
@@ -584,43 +607,34 @@ function AssetSortRow (props: { hideIcon: boolean, isSorted: boolean, assetSortT
       style={tailwind('px-4 flex flex-row justify-between pt-5')}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx:607]:
```suggestion
      style={tailwind('px-10 flex flex-row justify-between pt-5')}
```

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx [hunk 5]
```diff
@@ -361,13 +359,36 @@ export function PortfolioScreen ({ navigation }: Props): JSX.Element {
...
+                    dismissModal(true)
+                  }}
+                  style={tailwind('self-end pt-2.5')}
+                >
+                  <ThemedIcon
+                    dark={tailwind('text-mono-dark-v2-900')}
+                    light={tailwind('text-mono-light-v2-900')}
+                    iconType='Feather'
+                    name='x-circle'
+                    size={20}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx:381]:
```suggestion
                    size={24}
```

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx [hunk 6]
```diff
@@ -361,13 +359,36 @@ export function PortfolioScreen ({ navigation }: Props): JSX.Element {
...
+                  />
+                </ThemedTouchableOpacityV2>
+                <ThemedText
+                  style={tailwind('text-xl font-normal-v2 pt-0.5')}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx:385]: compensated by the "internal padding" of close icon
```suggestion
                  style={tailwind('text-xl font-normal-v2')}
```

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**